packages/labs/compiler/src/test/golden_compiler_test.ts

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
beautiful golden tests, you love to see 'em!

Reply:
You are the inspiration :D

Also no one wants to manually fix outputs after a compiler change!